searchads_api/api.py

```diff
@@ -9,8 +10,10 @@ class SearchAdsAPI:
     def __init__(
         self,
         org_id,
-        pem,
-        key,
+        pem_file_name=None,
```

Comment:
can we keep it to pem and key so we don't have to change all the scripts that count on this one @stepcha ?